GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -467,7 +475,7 @@ private ReturnCode RunAndRecordGVFSVerb<TVerb>(string archiveFolderPath, string
 
         private void FlushFilterLogBuffers()

```

Comment:
Rename to `FlushKernelDriverLogs` or just move this code inline since it is only called from one place?